### PR TITLE
Skuziv/homework/04

### PR DESCRIPTION
**Pull request by SerezhkaGit**

### CODE REVIEW

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+Console.WriteLine($"result: {c}, sum: {z}");
+//The last task
+int resultMaxValue = MaxValue(5, 9, 6);
+Console.WriteLine("MaxValue:" + resultMaxValue);
+int resultMinValue = MinValue(4, 2, 1);
+Console.WriteLine("MinValue:"+ resultMinValue);
+
+static int MaxValueReturn(int a, int b)
+{
+    if(a > b)
```
Comment (an-lev) [inline ConsoleApp1/Program.cs]: you can use either ternary operator or Math.Max(a,b)

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -1,9 +1,57 @@
...
+    sum = 0;
+    b = 2;
+    a = 2;
+    sum = a + b;
+    if (sum % 2 == 1) {  return true; }
+    return false; 
+}
+  int MaxValue(int a, int b, int c)
+{
+    if (a >= b && a >= c) { return a; }
```
Comment (an-lev) [inline ConsoleApp1/Program.cs]: you have already known how to find max value for 2 numbers, try to reuse it here

**Review by an-lev — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 2 above)
Comment (SerezhkaGit) [reply ConsoleApp1/Program.cs]: done

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (SerezhkaGit) [reply ConsoleApp1/Program.cs]: done

File: ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -1,9 +1,51 @@
...
+Console.WriteLine($"result: {c}, sum: {z}");
+//The last task
+int resultMaxValue = MaxValue(5, 9, 6);
+Console.WriteLine("MaxValue:" + resultMaxValue);
+int resultMinValue = MinValue(5, 9, 6);
+Console.WriteLine("MinValue:" + resultMinValue);
+
+static int MaxValueReturn(int a, int b)
+{
+    int c;
```
Comment (an-lev) [inline ConsoleApp1/Program.cs:21]: You can remove c variable and return value of ternary operator

File: ConsoleApp1/Program.cs [hunk 4]
```diff
@@ -1,9 +1,51 @@
...
+    a = 2;
+    sum = a + b;
+    if (sum % 2 == 1) {  return true; }
+    return false; 
+}
+  int MaxValue( int a,  int b, int c)
+{
+    int max1 = Math.Max(a, b);
+    int max2= Math.Max(max1, b);
+    return max2;
```
Comment (an-lev) [inline ConsoleApp1/Program.cs:44]: you can combine methods as `Math.Max(Math.Max(a, b), c)`

**Review by an-lev — APPROVED** [review]

**Pull request closed without merging**